app/javascript/vue/tasks/dwca_import/const/geographicalAreaDataOrigins.js

```diff
@@ -0,0 +1,12 @@
+export default {
+    NONE: '',
+    GEOGRAPHICAL_AREA_DATA_ORIGIN_GADM: 'gadm',
+    GEOGRAPHICAL_AREA_DATA_ORIGIN_SFG: 'SFG',
+    GEOGRAPHICAL_AREA_DATA_ORIGIN_NE_STATES: 'ne_states',
+    GEOGRAPHICAL_AREA_DATA_ORIGIN_COUNTRY_NAMES_AND_CODE_ELEMENTS: 'country_names_and_code_elements',
+    GEOGRAPHICAL_AREA_DATA_ORIGIN_NE_COUNTRIES: 'ne_countries',
+    GEOGRAPHICAL_AREA_DATA_ORIGIN_TDWG_L1: 'tdwg_l1',
```

Comment:
@gdower I think we might have to make these reference groups, like NE (states, countries), TDWG (all 4 levels). Does this make sense?

Reply:
I changed the GA scope to handle `tdwg` and `ne` and it sorts them so that the finest level available should be on top.